src/SprykerEco/Glue/UnzerRestApi/Controller/UnzerNotificationController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Glue\UnzerRestApi\Controller;
+
+use Generated\Shared\Transfer\RestUnzerNotificationAttributesTransfer;
+use Spryker\Glue\GlueApplication\Rest\JsonApi\RestResponseInterface;
+use Spryker\Glue\GlueApplication\Rest\Request\Data\RestRequestInterface;
+use Spryker\Glue\Kernel\Controller\AbstractController;
+
+/**
+ * @method \SprykerEco\Glue\UnzerRestApi\UnzerRestApiFactory getFactory()
+ */
+class UnzerNotificationController extends AbstractController
```

Comment:
```suggestion
class UnzerNotificationыController extends AbstractController
```
following the same logic
but class naming at Processor folder should be left in singular form

Reply:
Fixed.